setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
This is nothing to do with the pull request of printing omitted files - submit a new pull request.